betagouv/service-national-universel
# fix(admin, api): 1694 - Send new Brevo template when HTS switch to CL…

**Description**

Send new Brevo template when HTS switch to CLE having validated his phase 1.



**Checklist**

- [x] **⚠️ My code can have side-effects on other part of the code-base**
- [x] I have added the Plausible tags/events
- [x] I have performed a self-review of my code (and removed console.log)

**Ticket / Issue**

Closes [Notion: CLE - Récupérer l’attestation phase 1 pour un volontaire anciennement HTS validé passé en CLE](https://www.notion.so/jeveuxaider/CLE-R-cup-rer-l-attestation-phase-1-pour-un-volontaire-anciennement-HTS-valid-pass-en-CLE-2c8abbad2d494c6da28f86c021882a9c?d=ffa573c120d049d28415f6df095d7c80).

**Testing instructions**

1. Find a young HTS that has already validated it's `phase1`
2. Click on the pen icon next to the cohort name
3. Select an Etablissement, then a Classe
4. You should see the modal with the **Attestation Phase 1**
5. Hit next & validated
6. Check the requests in your network
